tests/test_conditional_steps.py

```diff
@@ -223,17 +223,38 @@ def test_prebuild_clean(test_env: ConditionalStepsTestEnv,
         test_env.check_fail(steps_info)
 
 
+def test_conditional_step_critical(test_env: ConditionalStepsTestEnv) -> None:
+    steps_info: StepsInfo = test_env.build_conditional_steps_info(is_conditional_step_passed=False)
+    steps_info.conditional_step.critical = True
+    test_env.check_success(steps_info)
+    assert test_env.captured_out.out
+    assert "WARNING" in test_env.captured_out.out
+
+
 # TODO: implement support of conditional step with children
 #  https://github.com/Samsung/Universum/issues/709
-def test_conditional_step_with_children(test_env: ConditionalStepsTestEnv) -> None:
+def test_conditional_step_with_children(test_env: ConditionalStepsTestEnv, capsys: pytest.CaptureFixture) -> None:
     steps_info: StepsInfo = test_env.build_conditional_steps_info(is_conditional_step_passed=True)
     steps_info.conditional_step.children = Configuration([Step(name=" dummy child 1"), Step(name=" dummy child 2")])
     test_env.check_fail(steps_info)
 
+    captured_out = capsys.readouterr()
+    assert "not supported" in captured_out.out
 
-def test_conditional_step_critical(test_env: ConditionalStepsTestEnv) -> None:
-    steps_info: StepsInfo = test_env.build_conditional_steps_info(is_conditional_step_passed=False)
-    steps_info.conditional_step.critical = True
-    test_env.check_success(steps_info)
-    assert test_env.captured_out.out
-    assert "WARNING" in test_env.captured_out.out
+
+# TODO: implement support of conditional steps addition and multiplication
+#  https://github.com/Samsung/Universum/issues/709
+def test_conditional_step_addition(test_env: ConditionalStepsTestEnv, capsys: pytest.CaptureFixture) -> None:
+    config_lines: List[str] = [
```

Comment:
Why do we need to store config as a list of lines instead of multiline text variable?

Reply:
Made similar to the `_write_config_file()` for uniformity.